Documentation/docs-site/content/usage/environments.en.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Environment"
+draft: false
+weight: 42
+---
+
+### Create an environment
+
+You can create an environment from node-env image and specify the resources such as memory and CPU along with poolsize. The poolsize controls how many warm pods are created initially.
```

Comment:
"You can create an environment on your cluster from an image for that language.  Optionally, you can specify CPU and memory resource limits.  You can also specify the number of initially pre-warmed pods, which is called the poolsize."